jax/experimental/jax2tf/examples/tflite/mnist/README.md

```diff
@@ -0,0 +1,92 @@
+# MNIST classification
+
+This directory contains the code necessary to train a simple convolution network on the MNIST dataset, and subsequently convert it using jax2tf into a model that can be converted to tflite format.
+
+This example is based on the [TensorFlow Lite demo of building a handwritten digit classifier app with TensorFlow Lite](https://developer.android.com/codelabs/digit-classifier-tflite). The model code is taken from [FLAX Linen Mnist Example](https://github.com/google/flax/tree/master/linen_examples/mnist).
+
+## Requirements
+
+* TensorFlow dataset mnist will be downloaded and prepared automatically, if necessary
+
+## Training the model
+
+You can train and export the model to tflite format yourself running
+```bash
+$ python mnist.py
+```
+
+The training itself takes only about 1 minute, assuming the dataset has already been downloaded. The dataset will be downloaded directly into a data/ directory in the /tmp/jax2tf/mnist directory. The training loop runs for 3 epochs, and the model is then converted to TensorFLow Lite and subsequently saved into /tmp/jax2tf/mnist/mnist.tflite.
```

Comment:
Suggestion:
- Add "for demonstration purposes" since 3 loops are probably not enough, but I don't know 🤷‍♂️ 

```suggestion
The training itself should take about 1 minute, assuming the dataset has already been downloaded. The dataset is loaded directly into a `data/` directory under `/tmp/jax2tf/mnist`. 

This example's training loop runs for 3 epochs for demonstration purposes. After training, the model is converted to SavedModels and then — TF Lite. In the end, it is saved as `mnist.tflite` under `/tmp/jax2tf/mnist/`.
```

WDYT? @qiuminxu